### PR TITLE
m2e updates

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: build/parent/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <sapphire-site>http://download.eclipse.org/sapphire/8.2/repository/</sapphire-site>
         <ivy-site>https://dist.apache.org/repos/dist/release/ant/ivyde/updatesite/ivy-2.4.0.final_20141213170938/</ivy-site>
         <ivyde-site>https://dist.apache.org/repos/dist/release/ant/ivyde/updatesite/ivyde-2.2.0.final-201311091524-RELEASE/</ivyde-site>
-        <m2e-site>http://download.eclipse.org/technology/m2e/releases/1.6/1.6.0.20150526-2032/</m2e-site>
+        <m2e-site>http://download.eclipse.org/technology/m2e/milestones/1.6/1.6.2.20150902-0001/</m2e-site>
         <m2e-wtp-site>http://download.eclipse.org/m2e-wtp/releases/mars/1.2/1.2.0.20150602-1740/</m2e-wtp-site>
```
Comment (gamerson) [inline build/parent/pom.xml]: Can you also search the latest m2e-wtp release and see if there is a 1.2.0.20159XXX release as well?

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Merged, thanks! See 57f7af7...458adf4
57f7af7...458adf4 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: build/parent/pom.xml (hunk 1 above)
Comment (jtydhr88) [inline build/parent/pom.xml]: I have researched and I didn't found any new version for m2e-wtp

Comment (gamerson) [inline build/parent/pom.xml]: Sounds good.

On Fri, Sep 11, 2015 at 1:47 AM, Terry Jia <notifications@github.com> wrote:

> In build/parent/pom.xml
> <https://github.com/gamerson/liferay-ide/pull/1134#discussion_r39245063>:
>
> > @@ -35,7 +35,7 @@
> >          <sapphire-site>http://download.eclipse.org/sapphire/8.2/repository/</sapphire-site>
> >          <ivy-site>https://dist.apache.org/repos/dist/release/ant/ivyde/updatesite/ivy-2.4.0.final_20141213170938/</ivy-site>
> >          <ivyde-site>https://dist.apache.org/repos/dist/release/ant/ivyde/updatesite/ivyde-2.2.0.final-201311091524-RELEASE/</ivyde-site>
> > -        <m2e-site>http://download.eclipse.org/technology/m2e/releases/1.6/1.6.0.20150526-2032/</m2e-site>
> > +        <m2e-site>http://download.eclipse.org/technology/m2e/milestones/1.6/1.6.2.20150902-0001/</m2e-site>
> >          <m2e-wtp-site>http://download.eclipse.org/m2e-wtp/releases/mars/1.2/1.2.0.20150602-1740/</m2e-wtp-site>
>
> I have researched and I didn't found any new version for m2e-wtp
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/1134/files#r39245063>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Pull request closed without merging**